teams/wg-rustc-dev-guide.toml

```
@@ -4,7 +4,19 @@ kind = "working-group"
 
 [people]
 leads = ["spastorino", "jyn514"]
-members = ["spastorino", "amanjeev", "tshepang", "togiberlin", "JohnTitor", "chrissimpkins", "LeSeulArtichaut", "jyn514", "camelid", "igaray"]
+members = [
+    "spastorino",
+    "amanjeev",
+    "tshepang",
+    "togiberlin",
+    "JohnTitor",
+    "chrissimpkins",
+    "LeSeulArtichaut",
+    "jyn514",
+    "camelid",
+    "igaray",
+    "rylev"
```

Since you split it across multiple lines, it might be nice to add a trailing comma so the diffs are easier to read in the future:

```suggestion
    "rylev",
```